drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -179,6 +184,42 @@ def max_score(self):  # pylint: disable=no-self-use
         """
         return 1
 
+    def get_score(self):
+        """
+        Return the problem's current score as raw values.
+        """
+        return Score(self.grade, self.max_score())
```

Comment:
Is there going to be a problem when existing blocks (already answered problems) that are storing a weighted grade in self.grade get processed using this code, and compare the weighted grade to max_score()?